Superbuild/ITKExternal.cmake

```diff
@@ -86,12 +89,12 @@ IF(BUILD_MOSAIC_TOOLS)
   )
 ENDIF()
 
-SET(itk_GIT_TAG "origin/itk5.1.0")
+SET(itk_GIT_TAG "enableH5HL")
```

Comment:
I just merged the commit from H5HL branch into master, so this should be changed to `master` until v5.2.0 is tagged.